reviews/models.py

```diff
@@ -88,6 +88,8 @@ def update_go(self):
                 # On NO-GO, reset the Proposal status
                 # TODO: also send e-mail?
                 else:
+                    from .utils import notify_supervisor_nogo
```

Comment:
Why the local import?

Reply:
This prevents a circular. I'd rather import models at the top layer in `utils.py` than import a utility at top level elsewhere. Models make more sense to have available everywhere.

Reply:
Right.

Friday 5PM  strikes again, should've spotted that